SocialCareCaseViewerApi/V1/Gateways/DatabaseGateway.cs

```diff
@@ -711,7 +714,7 @@ public CreateAllocationResponse CreateAllocation(CreateAllocationRequest request
                     Timestamp = dt.ToString("dd/MM/yyyy H:mm:ss"), //in line with imported form data
                     WorkerEmail = allocatedBy.Email,
                     Note =
-                        $"{dt.ToShortDateString()} | Allocation | {worker.FirstName} {worker.LastName} in {team.Name} was allocated to this person (by {allocatedBy.FirstName} {allocatedBy.LastName})",
+                        $"{dt.ToShortDateString()} | Allocation | {person.FirstName} {person.LastName} was allocated to the team {team.Name} (by {allocatedBy.FirstName} {allocatedBy.LastName})",
```

Comment:
What about wording for other scenarios? For example what happens when the person is allocated to a worker and not a team? That would require different note.

Reply:
I believe we are not allowing those scenarios moving further.

Reply:
Added optional worker insert